its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -70,6 +72,21 @@ public class TestUtils {
   private static final String MSBUILD_PATH = "MSBUILD_PATH";
   private static final String MSBUILD_PATH_DEFAULT = "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\msbuild.exe";
 
+  public static OrchestratorBuilder prepareOrchestrator() {
+    return Orchestrator.builderEnv()
+      .useDefaultAdminCredentialsForBuilds(true)
+      // Versions of SonarQube and plugins support aliases:
+      // - "DEV" for the latest build of master that passed QA
+      // - "DEV[1.0]" for the latest build that passed QA of series 1.0.x
+      // - "LATEST_RELEASE" for the latest release
+      // - "LATEST_RELEASE[1.0]" for latest release of series 1.0.x
+      // The SonarQube alias "LTS" has been dropped. An alternative is "LATEST_RELEASE[9.9]".
+      // The term "latest" refers to the highest version number, not the most recently published version.
```

Comment:
I don't understand this comment (what latest means.)

Reply:
Probably something like that version 9.9.0.1002 could have been published later than 9.9.1003.

I've replaced the whole thing with link to the orchestrator readme instead. That one is more up to date.